### DISCUSSION TITLE
[Feature Request] Queries in the Alert List panel

### DISCUSSION THREAD

**Discussion by AbelThorne** [grafana/grafana#48969] [Dashboards, Panels & Visualizations]
The Alert Panel is great but it lacks flexibility. 
When one wants to display carefully selected alerts, it is much more powerful to use a Table Panel and queries on the ALERTS/ALERTS_ON_STATE metrics. But then, there is no way to benefit from the great UI of the Alert List panel (labels displayed  as colored tags, possibility to unfold to read the alert description and have a link to the runbook).
I would really love to see something that gets the best from both worlds: a fully queriable Alert List panel

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
